maestrowf/datastructures/core/parameters.py

```diff
@@ -299,7 +299,10 @@ def get_combinations(self):
             combo = Combination()
             for key in self.parameters.keys():
                 pvalue = self.parameters[key][i]
-                tlabel = self.labels[key].replace(self.label_token,
+                try:
+                    tlabel = self.labels[key][i]
+                except:
```

Comment:
`flake8` flags bare `except` statements as a style violation. I suggest explicitly checking for types here.